run.py

```diff
@@ -35,6 +35,7 @@
 # Commands, etc
 bot.load_extension("bot.cogs.snakes")
 
-bot.run(os.environ.get("BOT_TOKEN"))
+#bot.run(os.environ.get("BOT_TOKEN"))
+bot.run('NDE0MDk5NTIyOTMxNzg1NzI4.DZiQjA.3eA2gcYy6D8KI8IhSfCr87oDePc')
```

Comment:
You've committed your bot token to the repository. Go to you bot's application page, invalidate the token, and then read over the bot setup guide again - add your token to the `.env` file like we described, and return this line to how it was when we gave it to you.